float64 x
float64 y
float64 z
---
float64 latitude
float64 longitude
float64 altitude
bool success
string message
